library/colour.cpp

```diff
@@ -46,14 +46,14 @@ Colour Colour::FromHex(const UString &hexcode)
 		return {0, 0, 0};
 	}
 	// invalid characters
-	else if (std::any_of(++hexcode_lower.begin(), hexcode_lower.end(), isxdigit))
+	else if (std::none_of(++hexcode_lower.begin(), hexcode_lower.end(), isxdigit))
```

Comment:
Was this a bug that made it through the last PR? If this is intended probably best to pull it out to a new PR, as it's not really "tooltip"-specific